### DISCUSSION TITLE
Remove Default Contact Points and Notification Policies

### DISCUSSION THREAD

**Discussion by saitbai** [grafana/grafana#69466] [Alerting]
Hi,

Please remove default contact points and notification policies, It does prevents provisioning from yaml file and has no usage.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82639. Feel free to continue the discussion around this there. Thank you!
